tests/beacon/types/test_validator_record.py

```diff
@@ -7,3 +10,26 @@ def test_defaults(sample_validator_record_params):
     validator = ValidatorRecord(**sample_validator_record_params)
     assert validator.pubkey == sample_validator_record_params['pubkey']
     assert validator.withdrawal_credentials == sample_validator_record_params['withdrawal_credentials']  # noqa: E501
+
+
+def test_is_active(sample_validator_record_params):
```

Comment:
Favor using parametrize for code reuse.

`status, is_active`

Reply:
Resolved. Thanks.